tuning/config/peft_config.py

```diff
@@ -1,9 +1,8 @@
 from dataclasses import dataclass, field
 from typing import List
 
-
 @dataclass
-class lora_config:
+class LoraConfig:
```

Comment:
dataclass names have to be camel case as per Python naming conventions. I want to add a black formatter to this repo going forward, and it would fail without this change. To handle name resolution with HF peft, I have renamed the file to peft_config. In code we would refer to it as `peft_config.LoraConfig`
If there is a need to further to distinguish the two in future, we can rename them as `CustomLoraConfig`